index.html

```diff
@@ -0,0 +1,94 @@
+      <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/fontawesome.css" integrity="sha384-jLKHWM3JRmfMU0A5x5AkjWkw/EYfGUAGagvnfryNV3F9VqM98XiIH7VBGVoxVSc7" crossorigin="anonymous"/>
+			<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/regular.min.css" integrity="sha512-k2UAKyvfA7Xd/6FrOv5SG4Qr9h4p2oaeshXF99WO3zIpCsgTJ3YZELDK0gHdlJE5ls+Mbd5HL50b458z3meB/Q==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+		</head>
+		<body>
+			<main>
+				<div class="header-background-mobile"></div>
+				<nav id="NavBar-status-light">
+					<time id="Time-Style">
+						<time class="Time">9:45</time>
+					</time>
+						<i class="fa-solid fa-wifi fa-xs"></i>	
+					<div class="sig-wf-bat">
+						<div class="signals">
+							<i class="fa-duotone fa-signal-bars-weak"></i>
+							<i class="fa-duotone fa-signal-bars-fair"></i>
+							<i class="fa-duotone fa-signal-bars-good"></i>
+							<i class="fa-duotone fa-signal-bars-strong"></i>
+						</div>
+						<div class="Battery">
+							<span class="Capacity"></span>
+							<span class="Border"></span>
+							<span class="Cap"></span>
+						</div>
+						<div class="wifi" >
+							<i class="fa-duotone fa-wifi-weak"></i>
+						</div>
+					</div>
+					</div>
+					<span class="rectangle"></span>
+				</nav>
+				<header id="AppBar-Compact-Dark-iOS">
+					<div id="Toolbar-Dark-Compact">
+						<div class="Normal-Buttn-Tert-Enabled">
+								<div id="NavIcon-Menu" >
+									<div onclick="openNav()"><i class="fa-solid fa-bars fa-xl"></i></div>
+								</div>
+						</div>
+					</div>
+					<div class="Logo-Placeholder">
+						<div class="MyLogo">My Logo</div>
+					</div>
+				</header>
+				<div class="Headlines">
+					<div class="pry-Txt-Headlines">
+						<p>I’m Adeyemi
+							Glad to see you!</p>
+					</div>
+						<p class="pry-Txt-Body">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+					<div class="social">
+						<div class="Supportng-Text">
+							<p class="connect">LET'S CONNECT</p>
+						</div>
+					</div>
+					<div class="social-media-icons">
+						<ul>
+							<li><a href="twitter.com/McEmarkees.com"><i class="fa-brands fa-twitter fa-xl"></i></a></li>
+							<li><a href="Linkedin.com/emarkees.com"><i class="fa-brands fa-linkedin fa-xl"></i></a></li>
+							<li><a href="github.com/emarkees.com"><i class="fa-brands fa-github fa-xl"></i></a></li>
+							<li><a href="github.com/emarkees.com"><i class="fab fa-medium fa-xl"></i></a></li>
+							<li><a href="anglrlist.com"><i class="fa-brands fa-angellist fa-xl"></i></a></li>
+						</ul>
```

Comment:
- [ ] Your social media icons and your text `LET'S CONNECT` are overlapping. Kindly make sure to fix that.👌

#### See 👇️
![Screenshot from 2022-12-01 23-41-08](https://user-images.githubusercontent.com/89228743/205182349-0002c017-591d-4b9c-b935-390714c8eb22.png)